appveyor.yml

```diff
@@ -29,7 +29,7 @@ before_test:
   - bundle --version
 
 test_script:
-  - bundle exec rspec --backtrace
+  - chcp 65001 && bundle exec rspec --backtrace
```

Comment:
Do we want to do that? Seems that we want to test we work in non UTF-8 environments...

Reply:
But for me, if you don't set the correct local it will fail for every representation of strings with Japanese characters. Maybe I can run the command with Japanese locale https://docs.microsoft.com/en-us/windows/desktop/intl/code-page-identifiers to validate the correct behavior and also try to see what is the default local for appveyor.

Reply:
My concern here is that we want to work with the console mode being japanese, not force it to utf-8

Reply:
Sorry for the delay @JonRowe. I am thinking everyday about this open PR and other things I have to do on RSpec repos. 😥 

----

I [ran a build](https://ci.appveyor.com/project/rspec/rspec-core/builds/20546210/job/wrjfy6eel8ftw3h9) with `chcp` to display the default encoding of app veyor machine.
```
chcp
Active code page: 437
```

437 is for "United States". - [source](https://ss64.com/nt/chcp.html)

So I think by doing this we are going to the right direction. 437 encoding is probably not meant to display unicode char we have to change it. 
Related StackOverFlow answer https://stackoverflow.com/a/3781095/2747638
or https://superuser.com/questions/625866/cmd-how-to-use-non-english-characters

Reply:
I can drop the last commit when you are ok with the existing work.

Reply:
Alright, I honestly don't know any better so lets drop that commit and merge this, get a release out and see what affect this makes for our friends with problems